pipelines/build/common/build_base_file.groovy

```diff
@@ -1034,6 +1034,19 @@ class Builder implements Serializable {
             if (publish || release) {
                 if (release) {
                     context.println 'NOT PUBLISHING RELEASE AUTOMATICALLY, PLEASE SEE THE RERUN RELEASE PUBLISH BINARIES LINKS'
+                    releaseSummary.appendText('</ul>', false)
+                    releaseSummary.appendText("<b>TAPs COLLECTION and REALEASE:</b><ul>")
+                    def urlJobName = URLEncoder.encode("${env.JOB_NAME}", 'UTF-8')
+                    def tapCollectionUrl = "${context.HUDSON_URL}job/TAP_Collection/parambuild?Release_PipelineJob_Name=${urlJobName}"
+                    releaseSummary.appendText("<li><a href=${tapCollectionUrl}> RELEASE TAPs COLLECTION</a></li>")
+                    def String releaseToolUrl = "${context.HUDSON_URL}job/build-scripts/job/release/job/refactor_openjdk_release_tool/parambuild?RELEASE=${release}}&UPSTREAM_JOB_NAME=TAP_Collection&UPLOAD_TESTRESULTS_ONLY=true&dryrun=false"
+                    def tag = publishName
+                    tag = URLEncoder.encode(tag, 'UTF-8')
+                    def artifactsToCopy = '**/AQAvitTapFiles.tar.gz'
+                    artifactsToCopy = URLEncoder.encode(artifactsToCopy, 'UTF-8')
+                    def javaVersion=determineReleaseToolRepoVersion()
+                    releaseToolUrl += "&VERSION=${javaVersion}&TAG=${tag}&ARTIFACTS_TO_COPY=${artifactsToCopy}"
+                    releaseSummary.appendText("<li><a href=${releaseToolUrl}> RELEASE TEST RESULTS TAPs Link</a></li>")
```

Comment:
Please add a flag to control this. This will not work for Semeru.
- we do not use ci-jenkins-pipelines to trigger test builds anymore
- we do not have TAP_Collection job